### PR TITLE
add dashboard page and videos

### PR DESCRIPTION
**Pull request by ivanha09**
Added a dashboard overview page.

Added additional videos for:
* Issues page
* Dashboard page 
* Repository Settings page
* Closing all issues in Sider

Also reorganized it so that people could navigate with the menu on the righthand side.

### CODE REVIEW

**Comment by ivanha09** [comment]
I'm not sure how to fix the error with the relative links to anchor points. When I tried adding `.md` to the end of the anchor point I would get a warning from Docusaurus.

docusaurus_1  | [WARN] unresolved links in file 'getting-started/intro-videos.md' > [ '../getting-started/setup#video-tutorial.md',

File: website/static/css/custom.css [hunk 1]
```diff
@@ -21,28 +21,30 @@ ul.video-list {
...
 	min-height: 8rem
 }
 
 ul.video-list a .info>p,
 ul.video-list a .info>span {
-	color: #2E3B4D
+	color: var(--black)
 }
 
 ul.video-list a .info .title {
```
Comment (ybiquitous) [inline website/static/css/custom.css]: [question] It looks to me that all `.title` elements are removed. Is this CSS necessary yet?

**Comment by ybiquitous** [comment]
[suggestion] I suggest that the `.video-list` element's HTML and CSS can be refactored like below:

Keys:

- Apply [BEM](http://getbem.com/) naming. This helps us to understand it's one component.
- More simple CSS. Reduce unnecessary properties.

Code:

```html
<div class="VideoLink">
  <a class="VideoLink__link" href="../getting-started/setup#video-tutorial">
    <img class="VideoLink__thumb" src="https://img.youtube.com/vi/bCfgdf4cjcU/mqdefault.jpg" alt="" aria-hidden="true">
    <div>
      <p>Learn how to install Sider to automatically review your code.</p>
      <span class="VideoLink__duration">Duration 3 min</span>
    </div>
  </a>
</div>
```

```css
.VideoLink {
  /* Docusaurus needs a wrapper element. */
}

.VideoLink__link {
  display: flex;
  color: currentColor;
  padding: .5rem;
  border-radius: .25rem;
  position: relative;
}

.VideoLink__link:hover,
.VideoLink__link:focus {
  text-decoration: none;
  background: var(--white-grey);
}

.VideoLink__link:hover::after,
.VideoLink__link:focus::after {
  content: '';
  display: block;
  width: 4rem;
  height: 4rem;
  position: absolute;
  background: url("/assets/play-button.svg");
  background-size: 4rem;
  top: 2.75rem;
  left: 5.75rem;
}

.VideoLink__thumb {
  max-height: 8rem;
  margin-right: 2rem;
  border: 1px solid var(--light-grey);
}

.VideoLink__duration {
  color: #6e6e6e;
}
```

Demo:
[Image from Gyazo](https://gyazo.com/831bfc0dccd0836d3aafed4fbeb5ed41)

**Comment by ybiquitous** [comment]
To help review, I enabled `stylelint` on this repository's Sider settings. By default, Sider uses [`stylelint-config-recommended`](https://github.com/stylelint/stylelint-config-recommended), which reports possible errors (not stylistic issues)! 😉 

See https://help.sider.review/tools/css/stylelint#default-configuration

File: website/static/css/custom.css (hunk 1 above)
Comment (ivanha09) [reply website/static/css/custom.css]: Yes. Thanks. I'll remove it.

**Comment by ivanha09** [comment]
@ybiquitous Thanks for all the info and code. I've followed the steps and made the changes and made changes to the video iframe too.

**Comment by ybiquitous** [comment]
Nice work! 👍

File: docs/getting-started/intro-videos.md [hunk 2]
```diff
@@ -9,67 +9,130 @@ hide_title: true
...
-				<h2 class="title">Code Review with Phinder</h2>
-				<p class="description">Learn how to create a custom rule to search for code written in PHP.</p>
-				<span class="duration">Duration 3 min</span>
-			</div>
-		</a>
-	</li>
-</ul>
+## Starter Videos
+
+### [Getting Started with Sider](../getting-started/setup.md#video-tutorial)
```
Comment (ybiquitous) [inline docs/getting-started/intro-videos.md]: I find it now, links in heading does not work in the right-side navigation. See the movie below:

[Image from Gyazo](https://gyazo.com/6c65e1017605082d78206dc3fd375350)

https://deploy-preview-176--sider-docs.netlify.com/getting-started/intro-videos

### Expected

Jump to a heading on the same page.

### Actual

Jump to another page.

### Solution

I think links in headings are necessary. Instead, `.VideoLink` components do the same thing.

```diff
-### [Getting Started with Sider](../getting-started/setup.md#video-tutorial)
+### Getting Started with Sider
```

Comment (ivanha09) [reply docs/getting-started/intro-videos.md]: Thanks! I'll remove the links from the titles per your solution.

**Pull request merged**